gcc/rust/backend/rust-compile.cc

```diff
@@ -247,6 +247,8 @@ HIRCompileBase::compute_address_for_trait_item (
   // Algo:
   // check if there is an impl-item for this trait-item-ref first
   // else assert that the trait-item-ref has an implementation
+  //
+  // FIXME this does not support super traits
```

Comment:
Should this be turned into an issue?

Reply:
Its already an issue #914 so this PR fixes the type-resolution part but doesn't finish the code-generation part so this PR does not close that issue.